rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -30,6 +30,19 @@ exception AdminException {
   2: required AdminErrorCode errorCode,
 }
 
+struct AddDBRequest {
+  # the db to add. the db is added as slave, so upstream_ip must be provided
+  # if the db exists already, a DB_EXIST error is thrown
+  1: required string db_name,
+  2: required string upstream_ip,
+  # if overwrite is true, then we will clear any existing DB if there is already info
```

Comment:
can we replace "existing" with another word? since we have used it for the case where the process already manages the db. Here, we wanted to mean something else.